src/CMakeLists.txt

```diff
@@ -759,6 +759,7 @@ if( MIOPEN_BACKEND STREQUAL "OpenCL")
     target_link_libraries( MIOpen PUBLIC ${OPENCL_LIBRARIES} )
     list(APPEND PACKAGE_DEPENDS PACKAGE OpenCL)
 elseif(MIOPEN_BACKEND STREQUAL "HIPOC" OR MIOPEN_BACKEND STREQUAL "HIP")
+    target_link_libraries( MIOpen PRIVATE hiprtc)
```

Comment:
Enclose into `if(MIOPEN_USE_HIPRTC) ... endif()`

Reply:
Done

Reply:
@JehandadKhan @junliume Hmm... seems that not. 

And this `target_link_libraries` for hiprtc already exists but only for WIN32:
https://github.com/ROCm/MIOpen/blob/4033d2191107f8381c4022612ca896019997429a/src/CMakeLists.txt#L765-L767

Maybe we should just remove `AND WIN32`.

Please also notice that this PR links `hiprtc` but WIN32 code links `hiprtc::hiprtc`.

Reply:
@atamazov  updated.